modules/core/src/main/java/org/apache/ignite/cluster/ClusterMetrics.java

```diff
@@ -347,15 +347,15 @@
      * Gets percentage of time this node is busy executing jobs vs. idling.
      *
      * @return Percentage of time this node is busy (value is less than
-     *      or equal to {@code 1} and greater than or equal to {@code 0})
+     *      or equal to {@code 100} and greater than or equal to {@code 0})
      */
     public float getBusyTimePercentage();
 
     /**
      * Gets percentage of time this node is idling vs. executing jobs.
      *
      * @return Percentage of time this node is idle (value is less than
-     *      or equal to {@code 1} and greater than or equal to {@code 0})
+     *      or equal to {@code 100} and greater than or equal to {@code 0})
```

Comment:
You change the public contract with these changes. Let's leave this as is.

Reply:
It is not logical. Method name percentage of use, but to output the share